docs/src/views/CustomComponents.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react'
+import * as cx from 'classnames'
+import { NavLink } from 'react-router-dom'
+import { Header } from 'semantic-ui-react'
+import {
+  Button,
+  Divider,
+  Provider,
+  createComponent,
+  ComponentSlotStyle,
+  ComponentVariablesInput,
+} from '@stardust-ui/react'
+
+import DocPage from '../components/DocPage/DocPage'
+import ExampleSnippet from '../components/ExampleSnippet/ExampleSnippet'
+
+interface StyledButtonProps {
+  className?: string
+  styles?: ComponentSlotStyle
+  variables?: ComponentVariablesInput
+  children?: React.ReactNodeArray | React.ReactNode
+}
+
+const StyledButton: React.SFC<StyledButtonProps> = createComponent<StyledButtonProps, {}>({
+  displayName: 'StyledButton',
+  render({ stardust, className, children }) {
+    const { classes } = stardust
+    return <button className={cx(className, classes.root)}>{children}</button>
+  },
+})
+
+export default () => (
+  <DocPage title="Custom Components">
```

Comment:
maybe something like "Integrate Custom Components" - because currently this is not absolutely clear from header whether we are suggesting to **develop** custom components or **integrate** existing ones